test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -199,3 +228,45 @@ func runCpaSampleCode(ctx context.Context, f *framework.Framework, runTests int,
 	err = e2epod.WaitForPodSuccessInNamespaceTimeout(ctx, f.ClientSet, pod.ObjectMeta.Name, f.Namespace.Name, 300*time.Second)
 	gomega.Expect(err).To(gomega.BeNil(), utils.GetPodLogs(ctx, f, pod.ObjectMeta.Name, pod.Spec.Containers[0].Name))
 }
+
+func injectError(ctx context.Context, f *framework.Framework) {
+	yes := true
+	podSpec := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{Name: "qat-inject-error"},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:            "qat-inject-error",
+					Image:           imageutils.GetE2EImage(imageutils.BusyBox),
+					ImagePullPolicy: "IfNotPresent",
+					Command:         []string{"find", "/sys/kernel/debug/qat_*/heartbeat/", "-name", "inject_error", "-exec", "sh", "-c", "'echo 1 > {}'", "\\;"},
+					SecurityContext: &v1.SecurityContext{
+						Privileged: &yes,
+					},
```

Comment:
is this required?

Reply:
If my memory was right, then yeah. Let me test once more.

Reply:
It is not required. Let me remove.

Reply:
I thought it is not required, but it is required..! Without it, inject_error's script does not run.

Reply:
Looks like you have deleted some comments in this thread, adding back: "It is not required. Let me remove."

I don't need it when I'm testing with a simple `docker run`. What do you mean by "script does not run"?

Reply:
sorry for confusing. I was wrongly testing and thought it is not required. For 'writing' to `inject_error`, privileged is necessary.

Reply:
@eero-t this is very confusing to me also. I tested with all capabilities and without seccomp and I get "permission denied" also. The files are mounted OK.